source/gui/settingsDialogs.py

```diff
@@ -3028,8 +3028,9 @@ def makeSettings(self, settingsSizer: wx.BoxSizer) -> None:
 
 		self._appendSoundSplitModesList(sHelper)
 
-		# Translators: This is a label for the applications volume adjuster combo box in settings.
-		label = _("&Application volume adjuster status")
+		# Translators: This is a label for the
+		# "allow application volume adjustment" combo box in settings.
+		label = _("&Allow application volume adjustment:")
```

Comment:
A rewording suggestion to make this option clearer and in line with what you wrote in the User Guide.
* Indicate "NVDA" since it's the key point of this option: can NVDA play with the mixer?
* adjust -> modify: adjust seems suitable for volume commands but a bit less for mute command; modify seems a bit more general, thus suitable both for volume and mute commands.


```suggestion
		label = _("&Allow NVDA to modify the volume of other applications")
```

Reply:
I think Cyrille's suggestion is simpler and most users won't make this implication. Thoughts @Qchristensen ?

Perhaps "set" might be clearer

```suggestion
		label = _("&Allow NVDA to setthe volume of other applications")
```

Reply:
My impression is that "modify" seems more suitable for both the volume change and the mute command, whereas with "set" it seems less obvious that it also impacts the mute command.

It's only the personal opinion of a non native English speaker, so feel free to choose "set" or another wording.

Reply:
Personally, I would prefer modify as Cyrille suggested, or the original adjust, over set.
Or possibly "control".

```suggestion
		label = _("&Allow NVDA to control the volume of other applications:")
```

I do understand where @codeofdusk is coming from with audio ducking. I'm not sure users will make that assumption, though this point should be clarified somewhere if it isn't already.